src/synapso_cli/commands/init.py

```
@@ -0,0 +1,114 @@
+    _initialize_vector_store(config_file)
+    _initialize_chunk_store(config_file)
+
+
+def _initialize_meta_store(config_file: str):
+    try:
+        config: GlobalConfig = get_config(config_file)
+        meta_store_path = config.meta_store.meta_db_path
+        _initialize_sqlite_db(meta_store_path)
+    except Exception as e:
+        typer.echo(f"Error initializing meta store: {e}", err=True)
+        raise typer.Exit(1)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)